src/components/HidContacts.js

```diff
@@ -72,10 +86,9 @@ class HidContacts extends React.Component {
       <MaterialAsyncSelect
         isMulti={this.props.isMulti}
         loadOptions={this.getOptions}
-        getOptionLabel={(option) => {return option.name}}
```

Comment:
@barbaraReliefapps Why did you need these changes on HidContacts ?

Reply:
I was testing with the Contacts and since the HIDContact field was later deleted I forgot to remove my changes, modifying it in new commit